### PR TITLE
Add deploySC from bearby account

### PR DESCRIPTION
**Pull request by pivilartisant**

### CODE REVIEW

**Comment by pivilartisant** [comment]
the tests don't seem to like these imports: 
// import { DEPLOYER_BYTECODE } from '@massalabs/massa-web3/dist/esm/generated/deployer-bytecode';
// import * as StorageCost from '@massalabs/massa-web3/dist/esm/basicElements/storage';

With error
```Jest encountered an unexpected token```

File: jest.e2e.config.esm.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     '^.+\\.jsx?$': 'babel-jest',
   },
   transformIgnorePatterns: [
-    'node_modules/(?!(@massalabs/wallet-provider|@hicaru/bearby.js|big-varint)/)',
```
Comment (peterjah) [inline jest.e2e.config.esm.js:10]: why ?

File: src/bearbyWallet/BearbyAccount.ts [hunk 2]
```diff
@@ -30,6 +31,9 @@ import {
 import { networkInfos } from './utils/network';
 import { WalletName } from '../wallet';
 import isEqual from 'lodash.isequal';
+import { uint8ArrayToBase64 } from '../utils/argsToBase64';
+import { DEPLOYER_BYTECODE } from '@massalabs/massa-web3/dist/esm/generated/deployer-bytecode';
+import * as StorageCost from '@massalabs/massa-web3/dist/esm/basicElements/storage';
```
Comment (peterjah) [inline src/bearbyWallet/BearbyAccount.ts:36]: do not import things from dist

**Review by peterjah — CHANGES_REQUESTED** [review]

**Comment by pivilartisant** [comment]
blocked by: https://github.com/massalabs/massa-web3/pull/703

**Review by peterjah — CHANGES_REQUESTED** [review]

**Pull request closed without merging**